src/Row.php

```diff
@@ -192,8 +192,13 @@ protected function assertHas(string $col) : void
 
     protected function isModified(string $col) : bool
     {
-        $old = $this->init[$col];
-        $new = $this->cols[$col];
+        $old = is_bool($this->init[$col])
```

Comment:
Looks good to me, but without test… 😉

Reply:
@froschdesign note the new test -- thoughts?

Reply:
LGTM 👍